File: micronaut-mcp-server-java-sdk/src/test/java/io/micronaut/mcp/server/stateless/sync/tools/jsonschema/output/FenEvaluationResponse.java
package io.micronaut.mcp.server.stateless.sync.tools.jsonschema.output;

import io.micronaut.core.annotation.Introspected;
import org.jspecify.annotations.NonNull;
import io.micronaut.jsonschema.JsonSchema;
import jakarta.validation.constraints.NotBlank;

@Introspected
@JsonSchema
public record FenEvaluationResponse(
    @NonNull @NotBlank String fen,
    @NonNull @NotBlank String evaluation
) {
}

File: test-suite-docs-python/src/test/python/micronaut/mcp/docs/tools/jsonschema/output/FenEvaluationResponse.py
# tag::imports[]
from dataclasses import dataclass
from typing import Annotated

from jakarta.validation.constraints import NotBlank
from micronaut.jsonschema import JsonSchema
from micronaut.serde.annotation import Serdeable
from org.jspecify.annotations import NonNull
# end::imports[]


# tag::clazz[]
@Serdeable
@JsonSchema
@dataclass
class FenEvaluationResponse:
    fen: Annotated[str, NonNull, NotBlank]
    evaluation: Annotated[str, NonNull, NotBlank]
# end::clazz[]
